lib/generator/models.ex

```diff
@@ -0,0 +1,175 @@
+defmodule Thrift.Generator.Models do
+  def generate(path) do
+    schema = path
+    |> File.read!
+    |> Thrift.Parser.parse
+    |> Map.put(:refs, load_refs(path))
+
+    List.flatten([
+      generate_enums(schema),
+      generate_structs(schema),
+      generate_exceptions(schema),
+    ])
+    |> Enum.map(fn {name, quoted} ->
+      filename = name
+      |> inspect
+      |> String.split(".")
+      |> Enum.map(&Macro.underscore/1)
+      |> Enum.join("/")
+      |> Kernel.<>(".ex")
+      source = Macro.to_string(quoted)
+      {filename, source}
+    end)
+  end
+
+  def namespace(%{namespaces: namespaces}) do
+    case namespaces do
+      %{ex: %{path: path}} ->
+        path |> String.split(".") |> Enum.map(&Macro.camelize/1) |> Enum.join(".")
+      %{py: %{path: path}} ->
```

Comment:
It was a fallback to be able to scan our existing schemas repo without adding an "elixir" namespace to each one. I'll take it out.